pkg/webhooks/workload_webhook.go

```diff
@@ -208,28 +208,28 @@ func validatePodSetUpdates(acs *kueue.AdmissionCheckState, obj *kueue.Workload,
 	for i := range acs.PodSetUpdates {
 		psu := &acs.PodSetUpdates[i]
 		psuPath := basePath.Index(i)
-		if found := knowPodSets.Has(psu.Name); !found {
+		if !knowPodSets.Has(psu.Name) {
 			allErrs = append(allErrs, field.NotSupported(psuPath.Child("name"), psu.Name, knowPodSets.UnsortedList()))
```

Comment:
nit: I would suggest to return the list sorted. Returning unsorted could add something to remember about in integration tests.

Reply:
+1

We can use the `sets.List()` function.

Reply:
updated https://github.com/kubernetes-sigs/kueue/pull/1220/commits/b219249d415c88de69843728c446aadc55cc43ea